django_project/changes/models/sponsor.py

```diff
@@ -81,6 +87,12 @@ class Sponsor(models.Model):
         null=True,
         blank=True)
 
+    sponsorship_contact_email = models.CharField(
```

Comment:
Shouldn't this be an EmailField?

Reply:
Hi @cchristelis yes it should, I adopted the practise to use CharField as used for the other email field on the same model such as 
``` 
sponsor_email = models.CharField(
        help_text='Input an email of sponsor.',
        max_length=255,
        null=True,
        blank=True)
```

Reply:
please add email validators too like this one https://github.com/kartoza/projecta/blob/develop/django_project/certification/models/certifying_organisation.py#L92

Reply:
Also what's the difference between sponsorship_contact_email and sponsor_email ( https://github.com/kartoza/projecta/pull/661/files#diff-371604757162da387f77bda08ad26881L78 ) in this model?